torchdata/stateful_dataloader/__init__.py

```diff
@@ -4,19 +4,8 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from .incremental_state import (
-    _DATASET_ITER_STATE,
-    _DATASET_STATE,
-    _FETCHER_ENDED,
-    _FETCHER_STATE,
-    _flatten,
-    _IncrementalState,
-    _IncrementalWorkerState,
-    _Tombstone,
-    _unflatten,
-    _WORKER_ID,
-)
+from .sampler import BatchSampler, RandomSampler
```

Comment:
I pulled this out here because of the way we inherit from torch.utils.data.DataLoader, we need these to be patched in order for state_dict to work with them, and importing here is sufficient. Side effects should be negligble, and should get tested by test_dataloader.py tests